### PR TITLE
chore: update buildjet/cache

### PR DESCRIPTION
**Pull request by skjnldsv**
@SystemKeeper @susnux

### CODE REVIEW

**Review by SystemKeeper — APPROVED** [review]
Let's go!

File: workflow-templates/cypress.yml [hunk 1]
```diff
@@ -95,7 +95,7 @@ jobs:
 
     steps:
       - name: Restore context
-        uses: buildjet/cache/restore@e376f15c6ec6dc595375c78633174c7e5f92dc0e # v3
+        uses: buildjet/cache/save@v4
```
Comment (skjnldsv) [inline workflow-templates/cypress.yml:98]: Should have been restore :see_no_evil:

Comment (SystemKeeper) [reply workflow-templates/cypress.yml:98]: 🙈

File: workflow-templates/command-compile.yml [hunk 2]
```diff
@@ -94,7 +94,7 @@ jobs:
 
     steps:
       - name: Restore cached git repository
-        uses: buildjet/cache@e376f15c6ec6dc595375c78633174c7e5f92dc0e # v3
+        uses: buildjet/cache/save@v4
```
Comment (SystemKeeper) [inline workflow-templates/command-compile.yml:97]: I guess here it's also wrong?

Comment (skjnldsv) [reply workflow-templates/command-compile.yml:97]: Aaaand we didn't fix this one too :see_no_evil:

Comment (SystemKeeper) [reply workflow-templates/command-compile.yml:97]: 🙈

**Pull request merged**